extensions/amp-auto-ads/0.1/anchor-ad-strategy.js

```diff
@@ -21,6 +21,7 @@ import {user} from '../../../src/log';
 const TAG = 'amp-auto-ads';
 const STICKY_AD_TAG = 'amp-sticky-ad';
 const OPT_IN_STATUS_ANCHOR_ADS = 2;
```

Comment:
Let's make this an enum with proper typing

Reply:
Done.